male-seed-beetle/scripts/download-data.py

```diff
@@ -0,0 +1,25 @@
+"""Downloads a data file from a given URL.
+
+This is the most basic of the download data scripts. It requires that the url for
+the data is provided as an argument, and it assumes that a data-raw folder
+has been created containing a .gitignore file.
+"""
+
+from pathlib import Path
+
+import requests
+
+# testing male-seed-beetle returned with print(f'{script_dir}')
+resource_dir = Path(__file__).resolve().parent.parent
+
+# testing male-seed-beetle/data-raw returned with print(f'{folder_path}')
+folder_path = resource_dir / "data-raw"
+
+url = "https://datadryad.org/stash/downloads/file_stream/30903"
```

Comment:
This downloads a file that has 403 still.

Reply:
No problem, I'll move it to the other resource and make sure to pick a dataset which will allow us to download automatically.